chsdi/models/bod.py

```diff
@@ -86,6 +86,7 @@ class LayersConfig(Base):
     geojsonUrlit = Column('geojson_url_it', Text)
     geojsonUrlrm = Column('geojson_url_rm', Text)
     geojsonUrlen = Column('geojson_url_en', Text)
+    shop = Column('shop_option_arr', postgresql.ARRAY(Text))
```

Comment:
This array contains the available shop option for this layer. 
```sql
CONSTRAINT shop_option_check CHECK (shop_option_arr <@ '{mapsheet,commune,district,canton,rectangle,whole}'::text[])
```
It could be any (or none) of the options above, in whatever order you want.